JavaScript homeworks/homework-6.js

```diff
@@ -0,0 +1,193 @@
+//Loops task 1//
+let PersonDescription = ['name: John', 'age: 30', 'city: New-York'];
+
+let i = 0;
+let text = '';
+while (PersonDescription[i]) {
+    text += PersonDescription[i] + '\n';
+    i++;
+}
+
+console.log(text);
+
+/*
+name: John
+age: 30
+city: New-York 
+*/
+
+
+//Loops task 2//
+function keyInObject(obj, keyToFind) {
+    for (let key in obj) {
+        if (key == keyToFind) {
+            return true;
+        }
+    }
+    return false;
+}
+
+let person = {
+    name: 'John',
+    age: 30,
+    city: 'New-York'
+};
+
+let keyToFind = 'age';
+if (keyInObject(person, keyToFind)) {
+    console.log('true');
+} else {
+    console.log('false');
+}
+
+// true 
+
+
+//Loops task 3//
+function combineTheWords(obj) {
+    let keys = Object.keys(obj);
+
+    let combineValue = '';
+    let i = 0;
+
+    do {
+        let currentProp = keys[i];
+        combineValue += obj[currentProp];
+        i++;
+
+    } while (i < keys.length);
+
+    return combineValue
+}
+
+let obj = {
+    prop1: 'Hello',
+    prop2: 'World',
+    prop3: '!'
+}
+
+console.log(combineTheWords(obj));
+
+
+//Loops task 4//
+function countNumberOfVowels(string) {
+    let vowels = ['a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U'];
```

Comment:
no need to check the uppercase vowels. Better convert the incoming string to lowercase and compare it with lowercase vowels only

Reply:
added "string = string.toLowerCase();"